skimage/feature/texture.py

```diff
@@ -284,37 +284,42 @@ def graycoprops(P, prop='contrast'):
 
 
 def local_binary_pattern(image, P, R, method='default'):
-    """Gray scale and rotation invariant LBP (Local Binary Patterns).
+    """Compute the local binary patterns (LBP) of an image.
 
-    LBP is an invariant descriptor that can be used for texture classification.
+    LBP is typically used for texture classification.
```

Comment:
How about: "LBP is a rotation-invariant visual descriptor often used in texture classification."?

Reply:
> I see that rotation invariance is now no longer mentioned anywhere; can we add it back?

It is mentioned under the specific methods when it is the case :wink: 

> How about: "LBP is a rotation-invariant visual descriptor often used in texture classification."?

According to the different methods, it's not always rotation-invariant; worse, it is not in the 'default' case (I haven't checked the source code for this, admittedly, I revamped the docstring based on the not-so-well-written-but-understandable descriptions). So it would be misleading to write "rotation-invariant" for the general case; it seems to me (at first glance) that the default should be 'uniform' rather than 'default' (but this would be an API change).

Reply:
:eyes: Oooh, I was misinformed all these years!! Thanks for the correction.

But I'd still expand that sentence with the "visual descriptor" bit.